### PR TITLE
Odim-6783: Remove default subscription for each server add request

### PR DESCRIPTION
**Pull request by chauberahul1993**

### CODE REVIEW

File: svc-events/events/publishevttosubscriber.go [hunk 1]
```diff
@@ -153,25 +147,18 @@ func (e *ExternalInterfaces) PublishEventsToDestination(data interface{}) bool {
 		subscription, _ := e.GetEvtSubscriptions(searchKeyAgg)
 		aggregateSubscriptionList = append(aggregateSubscriptionList, subscription...)
 	}
-	err = json.Unmarshal([]byte(requestData), &message)
-	if err != nil {
+	if err = json.Unmarshal([]byte(requestData), &message); err != nil {
```
Comment (amar-shalgar) [inline svc-events/events/publishevttosubscriber.go:150]: can this be avoided

**Review by amar-shalgar — CHANGES_REQUESTED** [review]

File: svc-events/events/evtsubscription.go [hunk 2]
```diff
@@ -649,26 +648,29 @@ func (e *ExternalInterfaces) CreateDefaultEventSubscription(originResources, eve
...
-		errorMessage := "error while trying to save event subscription data: " + err.Error()
-		evcommon.GenErrorResponse(errorMessage, errResponse.InternalError, http.StatusInternalServerError,
-			[]interface{}{}, &resp)
-		l.Log.Error(errorMessage)
-		return resp
-	}
+
+	// Removed creation of default subscription for each server add, adding only single default subscription at time of deployment with subscriptionID 0
+
+	// subscriptionID := uuid.New().String()
```
Comment (kavyamani596) [inline svc-events/events/evtsubscription.go:654]: remove this fully

Comment (chauberahul1993) [reply svc-events/events/evtsubscription.go:654]: We are keeping this comment

**Review by kavyamani596 — APPROVED** [review]

File: svc-events/events/publishevttosubscriber.go (hunk 1 above)
Comment (chauberahul1993) [reply svc-events/events/publishevttosubscriber.go]: done

**Review by hemkapoor — APPROVED** [review]

**Review by sriteja-sabbineni — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]

**Pull request merged**